### ISSUE TITLE
[Bug]: iOS build fails with ‘Command line “app-store” is deprecated’

### ISSUE THREAD

**Issue by pthorp** [ionic-team/capacitor#7632]
### Capacitor Version

💊   Capacitor Doctor  💊 

Latest Dependencies:

  @capacitor/cli: 6.1.2
  @capacitor/core: 6.1.2
  @capacitor/android: 6.1.2
  @capacitor/ios: 6.1.2

Installed Dependencies:

  @capacitor/cli: 6.1.2
  @capacitor/android: 6.1.2
  @capacitor/core: 6.1.2
  @capacitor/ios: 6.1.2

[success] iOS looking great! 👌
[success] Android looking great! 👌

### Other API Details

```Shell
npm: 10.8.2
node: v22.3.0
```

### Platforms Affected

- [X] iOS
- [ ] Android
- [ ] Web

### Current Behavior

When performing an iOS build in the terminal, or as part of the CI process, an error is thrown:
```
> npx cap build ios

✔ Building xArchive in 27.11s
✖ Building IPA - failed!
[error] 2024-08-22 15:00:48.749 xcodebuild[67491:5632754] [MT] IDEDistribution: -[IDEDistributionLogging
        _createLoggingBundleAtPath:]: Created bundle at path "/var/folders/rt/24tdz11j2pz60hc02rsd7w3w9900gn/T/App_2024-08-22_15-00-48.732.xcdistributionlogs".
        2024-08-22 15:00:48.851 xcodebuild[67491:5632754] [MT] IDEDistribution: Command line name "app-store" is
        deprecated. Use "app-store-connect" instead.
```

### Expected Behavior

The generated the file with App store information should contain 'app-store-connect', rather than 'app-store'.

### Project Reproduction

-

**Comment by jcesarmobile** [comment]
is that the complete error?

I "reproduced" the problem, but turns out it was just a warning and my error was something else.

**Comment by pthorp** [comment]
Indeed, that was the full error from the console.  It was also the only error that was shown in any of the logs, in the **IDEDistribution.critical.log**, and the IPA file was not generated.

**Comment by jcesarmobile** [comment]
can you run `xcodebuild archive -archivePath App.xcarchive -exportArchive -exportOptionsPlist archive.plist -exportPath output -allowProvisioningUpdates -configuration Release` from `ios/App/` folder?

Are you using automatic or manual signing?

**Comment by pthorp** [comment]
The `archive.plist` file is not present in that folder, so the command fails with a `no such file` exception.

e.g. 
```
App > ls -l

24 Aug 11:26 App
28 Aug 17:44 App.xcodeproj
22 Aug 09:08 App.xcworkspace
24 Aug 11:26 Podfile
22 Aug 15:00 Podfile.lock
22 Aug 08:51 Pods
22 Aug 20:32 output
```

When this issue occurred, we had it set up with manual signing and choosing the required distribution profile.

**Comment by jcesarmobile** [comment]
`archive.plist` is generated by the `npx cap build ios` command.

Manual signing is not supported at the moment, that's what causing the build to fail.

You can track the feature request on https://github.com/ionic-team/capacitor/issues/7625

**Comment by pthorp** [comment]
OK, no problem, thank you for your feedback.

**Comment by jcesarmobile** [comment]
keeping it open to fix the warning, it will eventually break
